app/context-map/workload.js

```diff
@@ -0,0 +1,20 @@
+const Workload = require('../points/domain/workload')
+const mapToTiers = require('./tiers')
+const Locations = require('../../app/staging/constants/locations')
+
+module.exports = function (communityTierCounts, custodyTierCounts, licenseTierCounts) {
+  var custodyTiersObject = mapToTiers(Locations.CUSTODY,
+                                custodyTierCounts[0], custodyTierCounts[1], custodyTierCounts[2],
+                                custodyTierCounts[3], custodyTierCounts[4], custodyTierCounts[5],
+                                custodyTierCounts[6], custodyTierCounts[7])
+  var communityTiersObject = mapToTiers(Locations.COMMUNITY,
+                                  communityTierCounts[0], communityTierCounts[1], communityTierCounts[2],
+                                  communityTierCounts[4], communityTierCounts[4], communityTierCounts[5],
```

Comment:
Yea, good spot.

I wasnt sure if there was any other method of this, but I can take a google at it.